README.md

```diff
@@ -11,10 +11,6 @@ If you just want to use tket via Python, the easiest way is to install it with
 pip install pytket
 ```
 
-For full API documentation, as well as a comprehensive user manual and a
-selection of example notebooks, please follow the links from the
-[pytket](https://github.com/CQCL/pytket) main page.
```

Comment:
This is just removed and not replaced?

Reply:
This was outdated. It must have been a long time ago that the API docs were in thhe `pytket` repo.

I don't think its particularly important to point users to the `pytket-docs` repo. The content that is in there is in more readable form on the TKET website anyway.

I can add something describing the contents more accurately if you think that would be helpful.

Reply:
Updated the README to clarify

Reply:
Great, thank you!